test/manifest/encrypted.json

```diff
@@ -30,11 +33,21 @@
       "duration": 15,
       "key": {
         "method": "AES-128",
-        "uri": "https://priv.example.com/key.php?r=53"
+        "uri": "https://priv.example.com/key.php?r=53",
+        "iv": null
       },
       "uri": "http://media.example.com/fileSequence53-A.ts"
     },
     {
+      "duration": 14,
+      "key": {
+        "method": "AES-128",
+        "uri": "https://priv.example.com/key.php?r=54",
+        "iv": new Uint32Array([0, 0, 331, 3063767524])
```

Comment:
Ok, that makes sense. Any reason you went with `Uint32Array` instead of `Uint8Array` here? I'd be cool with pulling this in if you could change the file extension of the examples to `.js`. We'll just have to be strict about the sort of expressions we let in here in the future.